modules/ossm-rn-known-issues.adoc

```diff
@@ -15,30 +15,44 @@ Module included in the following assemblies:
 
 These limitations exist in {SMProductName}:
 
-* {SMProductName} does not yet support link:https://issues.redhat.com/browse/MAISTRA-1314[IPv6], as it is not yet fully supported by the upstream Istio project.  As a result, {SMProductName} does not support dual-stack clusters.
+* {SMProductName} does not yet fully support link:https://issues.redhat.com/browse/MAISTRA-1314[IPv6]. As a result, {SMProductName} does not support dual-stack clusters.
 
 * Graph layout - The layout for the Kiali graph can render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions). Because it is difficult if not impossible to create a single layout that renders nicely for every situation, Kiali offers a choice of several different layouts. To choose a different layout, you can choose a different *Layout Schema* from the *Graph Settings* menu.
 
 * The first time you access related services such as {JaegerShortName} and Grafana, from the Kiali console, you must accept the certificate and re-authenticate using your {product-title} login credentials. This happens due to an issue with how the framework displays embedded pages in the console.
 
 ifndef::openshift-rosa[]
-* The Bookinfo sample application cannot be installed on IBM Z and IBM Power.
+* The Bookinfo sample application cannot be installed on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 
-* WebAssembly extensions are not supported on IBM Z and IBM Power.
+* WebAssembly extensions are not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 
-* LuaJIT is not supported on IBM Power.
+* LuaJIT is not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 
+* Single stack IPv6 support is not available on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 endif::openshift-rosa[]
+
 [id="ossm-rn-known-issues-ossm_{context}"]
 == {SMProductShortName} known issues
 
 These are the known issues in {SMProductName}:
 
-* https://issues.redhat.com/browse/OSSM-2221[OSSM-2221] Gateway injection does not work in control plane namespace. If you use the Gateway injection feature to create a gateway in the same location as the control plane, the injection fails and OpenShift generates this message:
+* https://issues.redhat.com/browse/OSSM-3890[OSSM-3890] Attempting to use the Gateway API in a multitenant mesh deployment generates an error message similar to the following:
++
+`2023-05-02T15:20:42.541034Z	error	watch error in cluster Kubernetes: failed to list *v1alpha2.TLSRoute: the server could not find the requested resource (get tlsroutes.gateway.networking.k8s.io)
```

Comment:
This block is too long to fit inline and needs to be formatted as system message output per the guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#system-messages as `[source,text]`.